tests/cupy_tests/test_cudnn.py

```diff
@@ -343,3 +343,115 @@ def test_call(self):
         else:
             with self.assertRaises(self.err):
                 self.call()
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+    'ksize': [1, 3, 5],
+    'stride': [1, 2, 4],
+    'auto_tune': [True, False],
+    'layout': layouts
+}))
+@unittest.skipUnless(cudnn_enabled, 'cuDNN is not available')
+class TestConvolutionInvalid(unittest.TestCase):
```

Comment:
Could you write a comment under which circumstances it fails? That'd make maintenance easier.

Reply:
Got it. will add a comment later.